cps/kni.c

```diff
@@ -640,7 +849,20 @@ kni_cps_route_event(struct cps_config *cps_conf)
 			num_updates++;
 	} while (num_updates < MAX_CPS_ROUTE_UPDATES);
 
-	/* TODO Send updates to GK block to update LPM table. */
+	if (cps_conf->gk == NULL)
+		return;
```

Comment:
Please add a warning message here since there shouldn't be any rtnetlink message under this configuration.

Reply:
Done.